src/OpenTelemetry.Instrumentation.Runtime/RuntimeMetrics.cs

```diff
@@ -82,7 +84,7 @@ public RuntimeMetrics(RuntimeMetricsOptions options)
 
             if (options.IsProcessEnabled)
             {
-                this.meter.CreateObservableCounter("process.cpu.time", this.GetProcessorTimes, "s", "Processor time of this process");
+                this.meter.CreateObservableCounter("process.cpu.time", GetProcessorTimes, "ns", "Processor time of this process");
```

Comment:
I think this should be `s` instead of `ns`?

Reply:
Yes, I fixed it in the other PR. This PR was mainly about GC until I came to "optimize" the array allocation.

Reply:
Yes, it should be `s`.

Both the [system metrics](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/system-metrics.md#systemcpu---processor-metrics) and [process metrics](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/process-metrics.md#process) use `s` as unit for CPU time.

So does [Go](https://github.com/open-telemetry/opentelemetry-specification/pull/2392#issuecomment-1058546909).

I'll modify it back to `s`.